runtime/doc/eval.txt

```diff
@@ -4244,8 +4212,7 @@ Textlock							*textlock*
 In a few situations it is not allowed to change the text in the buffer, jump
 to another window and some other things that might confuse or break what Vim
 is currently doing.  This mostly applies to things that happen when Vim is
-actually doing something else.  For example, evaluating the 'balloonexpr' may
-happen any moment the mouse cursor is resting at some position.
+actually doing something else.
```

Comment:
Maybe there could be another example here.

Reply:
sure, how about 'formatexpr'